Vizzuality/trase
# Feature/full screen button

## Description

This PR adds a full-screen button only to the iframe contained application.
This button once pressed uses the Full-Screen API to expand the app to full size.

It's mandatory that the iframe has the allowfullscreen and allow="fullscreen" attributes

Caveat: The browser is expanded to full-screen mode. I don't think this was part of the requirement but I don't think there its another way with the Full-screen API.

## Testing instructions

- Run a server in the tests/iframe folder. 
- Go to the port that the server serves (in my case 8000). 
- Press the full-screen button: It should expand and change to the exit button. 
- It should exit the full screen when the exit button or ESC key is pressed.

![full](https://user-images.githubusercontent.com/9701591/98116372-79b24f80-1ea8-11eb-8ea6-e7437422957a.gif)
